src/scripting/game_lua_kernel.cpp

```diff
@@ -1670,8 +1670,8 @@ int game_lua_kernel::intf_end_turn(lua_State* L)
 	//note that next_player_number = 1, next_player_number = nteams+1 both set the next team to be the first team
 	//but the later will make the turn counter change aswell fire turn end events accoringly etc.
 	if (!lua_isnoneornil(L, 1)) {
-		int npn = luaL_checkinteger(L, 1);
-		if (npn <= 0 /*TODO: || npn > 2*nteams*/) {
+		int npn = luaL_checkinteger(L, 1), max = 2 * teams().size();
+		if (npn <= 0  || npn > max) {
```

Comment:
Why create the `max` variable, instead of just putting `2 * teams().size()` in the `if` statement?

Reply:
Because the latter caused a comparison of differently-signed integers warning, which the CI treats as an error.

Reply:
Either a `static_cast<unsigned>` or putting the `int max =` on its own line would be better than having one line with two declarations separated by a comma.

Reply:
I don't see what's wrong with having two declarations on the same line, but if it'll make you stop complaining, I'll put them on separate lines.